library/src/scala/Tuple.scala

```diff
@@ -298,6 +311,12 @@ sealed trait NonEmptyTuple extends Tuple {
   inline def tail[This >: this.type <: NonEmptyTuple]: Tail[This] =
     runtime.Tuples.tail(this).asInstanceOf[Tail[This]]
 
+  /** Given a tuple `(a1, ..., am)`, returns the reversed tuple `(am, ..., a1)`
+   * consisting all its elements.
+   */
+  @experimental
+  inline def reverse[This >: this.type <: Tuple]: Reverse[This] =
```

Comment:
This method should also be moved to the `Tuple` trait, currently it is in `NonEmptyTuple`